USF-2745 Document public utility functions in checkout dropin (#439)
commerce-docs/microsite-commerce-storefront: src/content/docs/dropins/checkout/index.mdx

These controls include forms to introduce required information for contact details like email address, delivery and billing addresses, shipping options, and payment methods. Established customers who added items to the cart as a guest have the ability to sign in, automatically loading default addresses and contact details.

## Available resources

> The checkout drop-in component provides several resources to help you implement and customize your checkout experience:

Reviewer: ```suggestion
The checkout drop‑in component includes the following resources to implement and customize checkout:
```

- **[API Functions](/dropins/checkout/functions/)** - Core functions for managing checkout operations like authentication, shipping methods, and order placement

> - **[Utility Functions](/dropins/checkout/utilities/)** - Helper functions moved from the integration layer for DOM manipulation, form handling, data transformation, and more

Reviewer: ```suggestion
- **[Utility Functions](/dropins/checkout/utilities/)** -Helper functions for DOM manipulation, form handling, data transforms, and more
```
